_jobs/2021-10-01-optimaldynamics.md

```diff
@@ -1,9 +1,10 @@
 ---
 hidden: false
-layout: base
+layout: job-detail
 name: "Optimal Dynamics"
 website: "https://boards.greenhouse.io/optimaldynamics/jobs/4034910004"
 website_text: "Apply here"
+post_photo_url:
```

Comment:
We might revisit this name later, but it shouldn't impact you. 

I suspect what we want is a link to the sponsor and then to pull their logo.